frontend/src/metabase/visualizations/components/Visualization/Visualization.jsx

```diff
@@ -538,7 +538,7 @@ Visualization.defaultProps = defaultProps;
 export default _.compose(
   ExplicitSize({
     selector: ".CardVisualization",
-    refreshMode: props => (props.isVisible ? "throttle" : "debounce"),
+    refreshMode: props => (props.isVisible ? "throttle" : "debounceLeading"),
```

Comment:
Uses similar approach described in https://github.com/metabase/metabase/pull/31963#diff-48d6f4802294f837678e90fda148a480099b0db885a58fb28347d64fb16c062cR109-R110. It makes sense to use debounceLeading instead debounce so that ExplicitSize will calculate size on the first render, not waiting after some wait time before calculating the size which caused the bug.